turbo/jsonrpc/eth_api.go

```diff
@@ -174,6 +175,8 @@ func (api *BaseAPI) chainConfig(ctx context.Context, tx kv.Tx) (*chain.Config, e
 		return cfg, err
 	}
 
+	cfg.UsingEthereumHardfork = api.usingEthHardfork
```

Comment:
worth double checking if we need this.  Afaik the chain config is cemented into the DB as part of initialising the DB on a fresh start, maybe we can have this property written this way rather than setting it here?   We might not need the field on the `api` this way.